yb-voyager/src/queryissue/queryissue.go

```diff
@@ -29,12 +31,15 @@ import (
 )
 
 type ParserIssueDetector struct {
+	SourceSchemaList []string
 	// TODO: Add fields here
 	// e.g. store composite types, etc. for future processing.
 }
 
-func NewParserIssueDetector() *ParserIssueDetector {
-	return &ParserIssueDetector{}
+func NewParserIssueDetector(schemaList string) *ParserIssueDetector {
+	return &ParserIssueDetector{
+		SourceSchemaList: strings.Split(schemaList, "|"),
```

Comment:
This split logic based on "|" is spread out everywhere. Consider having this logic in cmd package somewhere, and take input directly as a string array